library/blob.c

```diff
@@ -2126,6 +2150,9 @@ int eblob_write_prepare(struct eblob_backend *b, struct eblob_key *key,
 err_out_cleanup_wc:
 	eblob_write_control_cleanup(&wc);
 err_out_exit:
+	if (!write_to_base)
```

Comment:
maybe bad way to unlock backend

Reply:
The code is congruent enough: either (write_to_base == 0, no lock) or (write_to_base == 1, lock). Seems ok independently on any label jumps.

Reply:
ok